examples/index.html

```diff
@@ -666,6 +666,7 @@ <h2 id="examples_by_props_label">Examples By Properties and States</h2>
             <td>
               <ul>
                 <li><a href="accordion/accordion.html">Accordion</a></li>
+                <li><a href="checkbox/checkbox.html">Checkbox  (Two State)</a> (<abbr title="High Contrast Support">HC</abbr>)</li>
```

Comment:
@jongund 
Does this change mean that the index builder requires the attribute to be in a `th` element when listed in the roles, state, and props table? If yes, I wonder if that code should be a bit more forgiving?